capa/features/extractors/viv/insn.py

```diff
@@ -120,6 +120,27 @@ def extract_insn_api_features(fh: FunctionHandle, bb, ih: InsnHandle) -> Iterato
                 yield API(name[1:]), ih.address
             return
 
+        if imports == {}:
+            # no imports implies the binary was likely statically linked.
+            # therefore, we try to use the symbol's table to fetch the api names
```

Comment:
I think we should try to use the symtab whenever it is present, even if there is some dynamic linking. If that doesn't work for some reason, lets document it and then revert to the strategy here. But, I suspect we'll have more results by using the symtab whenever it is present.

Reply:
I agree. I'll make the adjustment.